src/components/Person.tsx

```diff
@@ -0,0 +1,61 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person as PersonType } from '../types';
+
+type Props = {
+  person: PersonType;
+  people: PersonType[];
+};
+
+export const Person: React.FC<Props> = ({ person, people }) => {
+  const { slugParam } = useParams();
+  const { name, sex, born, died, slug, motherName, fatherName } = person;
```

Comment:
if u have 3 or more parameters - move them to separate line
```suggestion
  const { 
    name, 
    sex, 
    born,
    died, 
    slug, 
    motherName,
    fatherName 
  } = person;
```